packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -357,8 +357,8 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 		document.head.appendChild(element);
 		element.appendChild(document.createTextNode(`
 			.joplin-tinymce .tox-editor-header {
-				padding-left: ${styles.leftExtraToolbarContainer.width + styles.leftExtraToolbarContainer.padding * 2}px;
-				padding-right: ${styles.rightExtraToolbarContainer.width + styles.rightExtraToolbarContainer.padding * 2}px;
+				padding-left: ${120 + styles.leftExtraToolbarContainer.padding * 2}px;
+				padding-right: ${120 + styles.rightExtraToolbarContainer.padding * 2}px;
```

Comment:
Currently, this always increases the size reserved for the toolbar button. Based on [comment](https://github.com/laurent22/joplin/pull/10055#issuecomment-1978581425), I think it would make more sense to change the size conditionally. (This might require adding a dependency to the `useEffect`).

Reply:
Thank you for the updated video! (I had been looking at the video attached under "After" in the PR description :) )